Dockerfile

```
@@ -0,0 +1,22 @@
+# build the app in a separate build container
+FROM node:lts-slim as build
```

I would never include Dockerfiles in my repos with moving tags, but rather use semver tags in combination with a digest, to provide the best stability and developer experience. As I am used to work with alpine images exclusively, I would also pick the alpine version for the builder image.

So I would pick this:
```suggestion
FROM node:16.13.1-alpine as build
```
and would let renovate pin the digest afterwards (which it should do with the current configuration)